docs/current_docs/cloud/572923-get-started.mdx

```diff
@@ -239,21 +272,21 @@ The *Run Details* page includes detailed status and duration metadata about the
 
 Learn more about the [Dagger Cloud user interface](./reference/741031-user-interface.mdx).
 
-## Step 4: Integrate the Dagger Cloud cache
+## Step 4: Use cache volumes with the experimental Dagger Cloud cache (Team plan)
 
 :::info
```

Comment:
I would swap the position of the info and the note (note first to maintain consistency with other pages)

Reply:
Done! I also turned all these "at the end of this step" notes back into :::info::: for consistency. I tried the :::note::: style, but it's too light to be super readable. We can address style later.